package.json

```diff
@@ -24,12 +24,14 @@
   "homepage": "https://libp2p.io",
   "license": "MIT",
   "engines": {
-    "node": ">=10.0.0",
+    "node": ">=14.0.0",
     "npm": ">=6.0.0"
   },
   "scripts": {
     "lint": "aegir lint",
-    "build": "aegir build",
+    "build": "npm run build:proto && npm run build:proto-types && aegir build --no-types",
+    "build:proto": "pbjs -t static-module -w commonjs -r libp2p-pubsub-discovery --force-number --no-verify --no-delimited --no-create --no-beautify --no-defaults --lint eslint-disable -o src/proto.js ./src/proto.proto",
```

Comment:
Should this be `peer.proto` instead of `proto.proto`?

Reply:
sure, given that is a peer discovery piece, we should probably call it that way